src/style.css

```diff
@@ -1,3 +1,82 @@
+:root {
+  --main-header-bg-color: #fff;
+  --main-hover-text-color: #00acdc;
+  --main-text-color: #000;
+}
+
 body {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  height: 60px;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)